scripts/listener.ts

```diff
@@ -1,11 +1,51 @@
 import * as yargs from 'yargs';
 
 import { spec as EdgewareSpec } from '@edgeware/node-types';
+import * as CloverSpec from '@clover-network/node-tpye';
 import {
   chainSupportedBy, IEventHandler, CWEvent, SubstrateEvents, MarlinEvents, MolochEvents, EventSupportingChains
 } from '../dist/index';
 
+
+const CloverSpec = {
```

Comment:
This should probably go in its own file, maybe make a sub-folder for external chain specs, unless you can just use the imported library instead.